src/main/kotlin/lotto/domain/Lottos.kt

```diff
@@ -0,0 +1,12 @@
+package lotto.domain
+
+class Lottos(manualLottoNumbers: List<LottoNumbers>, autoLottoNumbers: List<LottoNumbers>) {
```

Comment:
```suggestion
class Lottos(manualLottoNumbers: LottoNumbers, autoLottoNumbers: LottoNumbers) {
```
구현하신 LottoNumbers를 활용해보면 어떨까요?

Reply:
이 클래스는 data를 wrapping 하는 성격에 가까워보입니다.
그렇다면 data class를 활용해보면 어떨까요?